### ISSUE TITLE
Laravel 5.7 not compatible

### ISSUE THREAD

**Issue by yarbala** [squigg/azure-queue-laravel#9]
**Summary of issue**

createPayload method call not compatible with Laravel 5.7 createPayload signature. Data not passed to job. 

file: squigg/azure-queue-laravel/src/AzureQueue.php line 62

change

```php
public function push($job, $data = '', $queue = null)
{
    $this->pushRaw($this->createPayload($job, $data), $queue);
}
```

to 

```php
public function push($job, $data = '', $queue = null)
{
    $this->pushRaw($this->createPayload($job, $queue, $data), $queue);
}
```

also shoud be fixed in "later" function like this:

```php
    public function later($delay, $job, $data = '', $queue = null)
    {
        $payload = $this->createPayload($job, $queue, $data);

        $options = new CreateMessageOptions();
        $options->setVisibilityTimeoutInSeconds($this->secondsUntil($delay));

        $this->azure->createMessage($this->getQueue($queue), $payload, $options);
    }
```

**Software versions**
This Package: 
Laravel: 5.7 
PHP: 7.2
Operating System: Windows / Linux / MacOS

**Comment by judgej** [comment]
Is there a recommended fork or alternative package for later versions of Laravel?

**Comment by squigg** [comment]
Sorry I've been focusing on other non-Laravel projects recently.

I'll happily accept a PR for changes for this version and/or will try set aside some time to see what's needing changed for 5.8

**Comment by judgej** [comment]
We need this working in the next few days, so will supply a PR if not beaten to it.

**Comment by squigg** [comment]
Should be fixed in 5.7.1

**Comment by judgej** [comment]
Thanks, I'll have a play with it.
